### ISSUE TITLE
I don't think anything is working

### ISSUE THREAD

**Issue by xupaii** [littleweeb/Desktop#28]
If it loads my **Currently Airing**, which it can only manage on startup, it still won't display anything. If I try going back into the Currently Airing tab again, it's constantly loading it.

Whilst trying to **download** Plastic Memories,  all I'm seeing is the genres, synopsis, the trailer as well as "add to favourites". I tried downloading Fruits Basket (2019), and I was stuck on retrieving synonyms like before. Behind the loading, I could see the "File", "Size" etc columns, but nothing below them. 

Also, the **taskbar icon** is just a white rectangle. 

TL;DR Nothing is working for me (i.e all stuck loading), I don't know if I've done something wrong but I can't use it, at all.

![Screenshot_318](https://user-images.githubusercontent.com/50333027/62823521-98347880-bb89-11e9-8c9d-4d8b2820ab6e.png)

**Comment by viktorrko** [comment]
The application is broken for some time now, nothing you can do about it
until dev decides to update it.

Dňa so, 10. aug 2019, 17:13 xupaii <notifications@github.com> napísal(a):

**Comment by xupaii** [comment]
I see, that's a shame. Can't wait for it to be back up then
